vds_api_client/vds_api_base.py

```diff
@@ -366,6 +366,8 @@ def delete_rois_from_account(self, rois):
         rois: (list of int) or Rois
             collection of roi ids or an Rois object (filtered)
         """
+        warnings.warn("delete_rois_from_account is deprecated, delete method should be used instead",
+                      DeprecationWarning)
```

Comment:
No problem. It's about the exercise right? Thanks for adding the message.

Still I'd like you to tell the user two additional things in the deprecation message:
- this method will be removed after 2021-01-01
- invoke the `delete` method on `self.rois` or `self.rois[item]`

Ignore the checks failing, those are not important now

Reply:
Alright, I added those in to the warning message. Sorry for the delay.

Reply:
Perfect, thanks a lot